packages/core/src/Cardano/Address/DRepID.ts

```diff
@@ -98,3 +99,11 @@ DRepID.toCip129DRepID = (drepId: DRepID): DRepID => {
   const credential = DRepID.toCredential(drepId);
   return DRepID.cip129FromCredential(credential);
 };
+
+DRepID.toAddress = (drepId: DRepID): EnterpriseAddress | undefined => {
+  const credential = DRepID.toCredential(drepId);
+  return new Address({
+    paymentPart: credential,
+    type: credential.type === CredentialType.KeyHash ? AddressType.EnterpriseKey : AddressType.EnterpriseScript
```

Comment:
Scripts can be drep? 🤔

Reply:
Not sure tbh. It's more for code completeness 😅